plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -53,13 +55,43 @@ def __init__(self, repo=None, conduit=None, config=None):
         self.available_manifests = []
         self.available_blobs = []
 
+        # Unit keys, populated by v1_sync.GetMetadataStep
+        self.v1_available_units = []
+        # populated by v1_sync.GetMetadataStep
+        self.v1_tags = {}
+
         # Create a Repository object to interact with.
         self.index_repository = registry.V2Repository(
```

Comment:
I considered that, but concluded that minimizing code change was more valuable at the moment.